site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -0,0 +1,139 @@
+
+Alternatively, `support.chat.example` may request `retail.example` to embed JavaScript that's hosted on `support.chat.example` on `retail.example` pages. This introduces security risks, because it allows the `support.chat.example` script to have elevated privileges on `retail.example`, such as the ability to access authentication cookies.
+
+Example use cases for CHIPS include any scenarios where cross-site subresources require some notion of session or persistent state that is scoped to a user's activity on a single top-level site, such as:
+
++   Third-party chat embeds
++   Third-party map embeds
++   Subresource CDN load balancing
++   Headless CMS providers
++   Sandbox domains for serving untrusted user content (such as googleusercontent.com and githubusercontent.com)
++   Third-party CDNs that use cookies to serve content that's access-controlled based on the authentication status on the first-party site (for example, profile pictures on social media sites hosted on third-party CDNs)
++   Front-end frameworks that rely on remote APIs using cookies on their requests
+
+### How does it work?
+
+CHIPS introduces a new cookie attribute, `Partitioned`, to support cross-site cookies that are partitioned by top-level context.  
+
+Under this proposal, when a user visits site A and embedded content from site C sets a cookie with the `Partitioned `attribute, the cookie is saved in a partitioned jar designated only for cookies that the site C sets when it's embedded on site A. The browser would only send that cookie when the top-level site is A.   
+
+When the user visits a new site, for example site B, an embedded C frame would not receive the cookie that was set when C was embedded in site A. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/PODInBIXZrgGeUoFhipj.png", alt="ALT_TEXT_HERE", width="800", height="443" %}
+
+If a user visits site C as a top level website, the partitioned cookie that C set when it was embedded in A will not be sent in that request either.
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/FaQYaZyTsAxCm8GvvVc8.png", alt="ALT_TEXT_HERE", width="800", height="608" %}
+
+### Why it's important to opt into cookie partitioning
+
+As browsers are phasing out unpartitioned third-party cookies, a couple of other approaches to partitioning have been attempted.
+
+Firefox announced that they are [partitioning all third-party cookies by default](https://hacks.mozilla.org/2021/02/introducing-state-partitioning/) in their ETP Strict mode, so all cross-site cookies are partitioned by the top-level site.   
+However, partitioning cookies without a third-party opt-in can lead to unexpected bugs, since some third party services have built servers with the expectation of an unpartitioned third-party cookie.
```

Comment:
```suggestion
However, partitioning cookies without a third-party opt-in can lead to unexpected bugs, since some third-party services have built servers which expect an unpartitioned third-party cookie.
```